src/main/java/com/ericsson/ei/handlers/RMQHandler.java

```diff
@@ -185,4 +214,64 @@ private boolean isRMQCredentialsSet() {
         return bindingKeysWithoutWhitespace.split(",");
     }
+
+    /**
+     * This method is used to delete the bindings in rabbitMQ.
+     * By comparing the binding keys used in the properties and binding keys stored in mongoDB.
+     * newBindingKeysArray is the only binding keys array.
+     * AMQPBindingObjectList is entire list of bindings.
+     * Binding key which is not present in the current AMQPBindingObjectList gets deleted and removed from mongoDB.
+     * @return
+     */
+
+    private void deleteBindings(String[] newBindingKeysArray, List<Binding> AMQPBindingObjectList) {
+        // Creating BindingKeys Collection in mongoDB
+        ArrayList<String> allDocuments = mongoDBHandler.getAllDocuments(dataBaseName, collectionName);
+        ArrayList<String> existingBindingsData = new ArrayList<String>();
+        if (!allDocuments.isEmpty()) {
+            for (String bindings : allDocuments) {
+                JSONObject bindingObj = new JSONObject(bindings);
+                final String mongoDbBindingKey = bindingObj.getString("bindingKeys");
+                String queryString = "{\"bindingKeys\": /.*" + mongoDbBindingKey + "/}";
+                if (!Arrays.asList(newBindingKeysArray).contains(mongoDbBindingKey)) {
+                    String destinationDB = bindingObj.getString("destination");
+                    String exchangeDB = bindingObj.getString("exchange");
+                    // Binding the old binding key and removing from queue
+                    Binding b = new Binding(destinationDB, DestinationType.QUEUE, exchangeDB, mongoDbBindingKey, null);
+                    amqpAdmin = new RabbitAdmin(connectionFactory());
+                    amqpAdmin.removeBinding(b);
+                    // Removing binding document from mongoDB
+                    MongoStringQuery query = new MongoStringQuery(queryString);
+                    mongoDBHandler.dropDocument(dataBaseName, collectionName, query);
+                } else {
+                    // storing the existing key into an array.
+                    existingBindingsData.add(mongoDbBindingKey);
+                }
+            }
+        }
+        // To store the new binding key into the mongoDB.
+        storeNewBindingKeys(existingBindingsData, AMQPBindingObjectList);
+    }
+
+    /**
+     * This method is used to store the bindings of new binding key into mongoDB.
+     * @return
+     */
+
+    private void storeNewBindingKeys(ArrayList<String> existingBindingsData, List<Binding> AMQPBindingObjectList){
+    // comparing with the stored key and adding the new binding key into the mongoDB.
+       for(final Binding bindingKey:AMQPBindingObjectList){
+            if(existingBindingsData.contains(bindingKey.getRoutingKey())){
+                LOGGER.info("Binding already present in mongoDB");
+            }else{
+                    BasicDBObject document = new BasicDBObject();
+                    document.put("destination",bindingKey.getDestination());
```

Comment:
Can these keys be set as constants?  Mongo DB related constants are defined in the mongo package.